Firebase/Messaging/FIRMessagingRmq2PersistentStore.m

```diff
@@ -256,8 +273,17 @@ - (void)openDatabase:(NSString *)dbName {
   BOOL didOpenDatabase = YES;
   if (![fileManager fileExistsAtPath:path]) {
     // We've to separate between different versions here because of backwards compatbility issues.
-    if (sqlite3_open([path UTF8String], &_database) != SQLITE_OK) {
-      FIRMessaging_FAIL(@"%@ Could not open rmq database: %@", kFCMRmqStoreTag, path);
+    int result = sqlite3_open([path UTF8String], &_database);
```

Comment:
Since the next two blocks exactly match, probably should be factored into a function for code size and readability.

Reply:
Yeah, agreed. Just didn't want to change a larger block of code this close to release 😄